gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/mapper/diagnosticreport/DiagnosticReportMapper.java

```diff
@@ -126,7 +127,12 @@ private DiagnosticReport createDiagnosticReport(RCMRMT030101UKCompoundStatement
         final DiagnosticReport diagnosticReport = new DiagnosticReport();
         final String id = compoundStatement.getId().get(0).getRoot();
 
-        diagnosticReport.setMeta(generateMeta(META_PROFILE_URL_SUFFIX));
+        var meta = confidentialityService.createMetaAndAddSecurityIfConfidentialityCodesPresent(
+            META_PROFILE_URL_SUFFIX,
```

Comment:
While it's true, should we keep this `META_PROFILE_URL_SUFFIX` as `META_PROFILE` for now in terms of consistency, as it's `META_PROFILE` above? If not could we please change the `META_PROFILE` within ObservationMapper to `META_PROFILE_URL_SUFFIX`? 😄